shelley/chain-and-ledger/formal-spec/chain.tex

```diff
@@ -346,21 +454,27 @@ \subsection{New Epoch Transition}
       e = e_\ell + 1
       &
       \var{ru} \neq \Nothing
-      \\
-      (\wcard,~\wcard,~\wcard,~\wcard,~\var{irwd'})\leteq\var{ru}
       &
-      \var{irwd'} = \getIR{es}
+      (\Delta t,~\Delta r,~\var{rs},~\Delta f)\leteq\var{ru}
+      \\
+      \Delta t+~\Delta r+\left(\sum\limits_{\wcard\mapsto v\in\var{rs}}\right)v+\Delta f=0
```

Comment:
Typo: `v` should be inside the existing big parenthesis.

Reply:
indeed! I get that in whatever change to the spec we make next